doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_soc_binaries.rst

```diff
@@ -15,34 +15,34 @@ Updating the nRF54H20 SoC binaries
 **********************************
 
 You can update the nRF54H20 SoC binaries in two ways.
-Both methods require using the :file:`nordic_top.suit` envelope, which can be found inside the nRF54H20 SOC binaries ZIP file.
+Both methods require using the :file:`nordic_top.suit` envelope, available inside the nRF54H20 SoC binaries ZIP file in the folder.
```

Comment:
```suggestion
Both methods require using the :file:`nordic_top.suit` envelope, available inside the nRF54H20 SoC binaries ZIP file.
```
A folder has not been mentioned before on this page, so can not refer to "the folder". Alternatively, can make the reference more specific to clarify which folder is being referred to.